codec/src/test/java/io/netty/handler/codec/base64/Base64Test.java

```diff
@@ -169,4 +169,24 @@ public void testOverflowEncodedBufferSize() {
     public void testOverflowDecodedBufferSize() {
         assertEquals(1610612736, Base64.decodedBufferSize(Integer.MAX_VALUE));
     }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void decodingFailsOnInvalidInputByte() {
+        Base64.decode(Unpooled.wrappedBuffer("eHh4\u00BD".getBytes(CharsetUtil.ISO_8859_1)));
```

Comment:
Please ensure to release all the created ByteBuf instances in a finally block (true for all added tests)